src/ingest_validation_tools/directory_validator.py

```diff
@@ -22,27 +22,30 @@ def validate_directory(
         required_patterns, allowed_patterns = _get_required_allowed(schema_files)
         dependencies = _get_dependencies(schema_files)
     except KeyError as e:
-        raise DirectoryValidationErrors(f"Error finding patterns for {path}: {e}")
+        raise DirectoryValidationErrors(f"Error finding patterns for {','.join(paths)}: {e}")
```

Comment:
I think this will break since it's trying to join PosixPath objects rather than strings. Maybe something like `f"Error finding patterns for {','.join([str(path) for path in paths])}: {e}"`?

Reply:
Thanks for pointing this out - I knew I had missed one of these when I was fixing the error messages!